tools/coqdep_common.ml

```diff
@@ -498,7 +502,7 @@ let rec treat_file old_dirname old_name =
   let complete_name = file_name name dirname in
   let stat_res =
     try stat complete_name
-    with Unix_error(error, _, _) -> error_cannot_stat complete_name error
+    with Unix_error(error, _, _) -> raise (Cannot_stat_file (complete_name, error))
```

Comment:
Why this change?

Reply:
So as to have different messages depending on whether the file is on the command line or in a project file (see below `treat_file_command_line` and `treat_file_coq_project`).